src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -1,20 +1,30 @@
 package billgates.view.gui;
 
 import javax.swing.*;
+import javax.swing.event.MenuEvent;
+import javax.swing.event.MenuListener;
+import javax.swing.filechooser.FileNameExtensionFilter;
 import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.io.File;
 
-public class TopMenuBar extends JMenuBar {
+public class TopMenuBar extends JMenuBar{
 
     public static final int DEFAULT_HEIGHT = 23;
     private final BoxLayout layout = new BoxLayout(this, BoxLayout.X_AXIS);
     private final JMenu importMenu = new GeneralMenu("Import");
     private final JMenu settingsMenu = new GeneralMenu("Settings");
     private final JMenu helpMenu = new GeneralMenu("Help");
+    private final Color[] colors = new Color[]{new Color(240, 140, 170), Color.RED, Color.ORANGE,
```

Comment:
It looks like these are constants, so you can convert them to `static` variables. Also, don't forget to update their variable names to adhere constants naming convention: `COLORS`.